test/integration/scripting_spec.mjs

```diff
@@ -2510,4 +2510,42 @@ describe("Interaction", () => {
       );
     });
   });
+
+  describe("Calculate field value even if one callback throws", () => {
+    let pages;
+    let otherPages;
```

Comment:
`otherPages` is unused in this test and can be removed. I have created PR #18567 to remove it from the other tests where it's unused.